plugins/services/src/js/utils/TaskUtil.js

```diff
@@ -146,15 +146,6 @@ const TaskUtil = {
     }
 
     const nodeZoneName = node.getZoneName();
```

Comment:
Small detail, since we just return the zone name this could be `return node.getZoneName()`

Reply:
I agree, I'll fix that 👍